File: benchmarks/cpu/primes/primes.py
def is_prime(n):
    if n < 2:
        return False
    d = 2
    while d * d <= n:
        if n % d == 0:
            return False
        d += 1
    return True

count = 0
for n in range(2, 200001):
    if is_prime(n):
        count += 1
print(count)

File: benchmarks/cpu/primes/primes.go
package main

import "fmt"

func isPrime(n int) bool {
	if n < 2 {
		return false
	}
	d := 2
	for d*d <= n {
		if n%d == 0 {
			return false
		}
		d += 1
	}
	return true
}

func main() {
	count := 0
	for n := 2; n <= 200000; n++ {
		if isPrime(n) {
			count += 1
		}
	}
	fmt.Println(count)
}
